app/build.gradle

```diff
@@ -61,7 +61,7 @@ android {
 
     lint {
         disable "GoogleAppIndexingWarning", "ButtonStyle", "AlwaysShowAction",
-                "MissingTranslation", "MissingPrefix"
+                "MissingTranslation", "MissingQuantity", "MissingPrefix"
```

Comment:
Please don't suppress lint checks.

I know this check is annoying currently, but this is an issue that Weblate should fix: https://github.com/WeblateOrg/weblate/issues/7520

Unless you can lint to documentation proving that this lint check failing will never cause any type of crash, it really shouldn't be disabled.

Reply:
Dropped assuming this won't be a blocker for this PR

Reply:
Well, lint failing will be a blocked because without lint reviewing other MRs will be a lot harder.

I can mark it hacktoberfest-accepted but this Weblate issue is the only reason I haven't been updating Gradle and targetSdk yet.

Reply:
Dropped Gradle version upgrade for now then